example-gto/code/mlem/.mlem/config.yaml

```diff
@@ -0,0 +1,6 @@
+#core:
+#  state:
+#    uri: s3://gto-mlem-example/state
+docker:
+  git_rev: feature/new-state
```

Comment:
Q: what is this about?

Reply:
this tells docker builder (used in heroku deployment) to install mlem from this branch (instead of pypi). This can be removed once we release 0.3

Reply:
do we have / plan to have this branch in this demo repo though?

Reply:
We don't have `mlem` branch in `example-gto` repo yet. I'm going to add it once we release MLEM `0.3.0`, and remove this part of `config.yaml`. Adding it to the TODO.

Reply:
Discussed this with Oded and Mike. Decided that we can release it here as it is now, and update it once we release 0.3.0 MLEM version to PyPi.

Reply:
Can we wait with this PR and merge after 0.3.0 release but still upload this to `example-gto`?

Reply:
Ok, I've uploaded this to `iterative/example-gto`. AWS state is in place and deployment works as expected.